### PR TITLE
docs(readme): new extensions

### PR DESCRIPTION
**Pull request by PikinerLiliya**
Added new extensions to the Readme file

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -23,8 +23,11 @@ Tools for developing extensions.
 List of the most Awesome extensions made for Dynamic Content.
 ​
 - [dc-extension-localisation](https://github.com/amplience/dc-extension-localization) - Automatic translation of localized fields.
-- [dc-extension-product-selector](https://github.com/amplience/dc-extension-product-selector) - Search and select products from Salesforce Commerce Cloud or SAP Hybris.
+- [dc-extension-product-selector](https://github.com/amplience/dc-extension-product-selector) - Search and select products from Salesforce Commerce Cloud, SAP Hybris or Commercetools.
 - [dc-extension-di-transform](https://github.com/amplience/dc-extension-di-transform) - Set image transformations such as point of interest or cropping in Dynamic Content.
+- [dc-extension-brand-colours](https://github.com/amplience/dc-extension-brand-colours) - Pick colours relevant to your brand's identity from customizable colour groups.
+- [dc-extension-rich-text](https://github.com/amplience/dc-extension-rich-text) - Customizable rich text editor with an inline content chooser.
```
Comment (pixelscript) [inline readme.md:29]: Mix of British and North American spelling - suggest Customizable to Customisable

**Review by pixelscript — APPROVED** [review]

File: readme.md [hunk 2]
```diff
@@ -23,8 +23,11 @@ Tools for developing extensions.
 List of the most Awesome extensions made for Dynamic Content.
 ​
 - [dc-extension-localisation](https://github.com/amplience/dc-extension-localization) - Automatic translation of localized fields.
-- [dc-extension-product-selector](https://github.com/amplience/dc-extension-product-selector) - Search and select products from Salesforce Commerce Cloud or SAP Hybris.
+- [dc-extension-product-selector](https://github.com/amplience/dc-extension-product-selector) - Search and select products from Salesforce Commerce Cloud, SAP Hybris or Commercetools.
 - [dc-extension-di-transform](https://github.com/amplience/dc-extension-di-transform) - Set image transformations such as point of interest or cropping in Dynamic Content.
+- [dc-extension-brand-colours](https://github.com/amplience/dc-extension-brand-colours) - Pick colours relevant to your brand's identity from customisable colour groups.
```
Comment (osecluna) [inline readme.md]: Please change "customisable" to "customizable"

File: readme.md [hunk 3]
```diff
@@ -23,8 +23,11 @@ Tools for developing extensions.
...
+- [dc-extension-rich-text](https://github.com/amplience/dc-extension-rich-text) - Customisable rich text editor with an inline content chooser.
```
Comment (osecluna) [inline readme.md]: Please change "customisable" to "customizable"

**Review by osecluna — CHANGES_REQUESTED** [review]

**Review by osecluna — APPROVED** [review]

File: readme.md (hunk 3 above)
Comment (PikinerLiliya) [reply readme.md]: done

File: readme.md (hunk 2 above)
Comment (PikinerLiliya) [reply readme.md]: done

**Review by osecluna — APPROVED** [review]

**Pull request merged**